app-rails/local.env.example

```diff
@@ -26,9 +26,9 @@ AWS_DEFAULT_REGION=<FILL ME IN>
 # Auth
 ############################
 
-AWS_COGNITO_USER_POOL_ID=<FILL ME IN>
-AWS_COGNITO_CLIENT_ID=<FILL ME IN>
-AWS_COGNITO_CLIENT_SECRET=<FILL ME IN>
+COGNITO_USER_POOL_ID=<FILL ME IN>
```

Comment:
Did you also mean to remove `AWS` from line 23, in `AWS_BUCKET_NAME=<FILL ME IN>` because you refer to `ENV["BUCKET_NAME"] elsewhere?

Also, maybe take it out of the AWS Services section.

Reply:
Yes. Thank you!